RFCs: update geospatial RFC

otan opened the pull request:
Update geospatial RFC to be in line with updated offerings and commands.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/52958)
<!-- Reviewable:end -->

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/52958#-:-MF0mkUw0fu_2N3IitHW:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @otan)

___
*[docs/RFCS/20200421_geospatial.md, line 606 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/52958#-MF0lbWn057efgbgV9g1:-MF0lbWn057efgbgV9g2:bbcl768) ([raw file](https://github.com/cockroachdb/cockroach/blob/2a693e9ea985ca1b128be4a72e75e3345bb977b4/docs/RFCS/20200421_geospatial.md#L606)):*
> ```
> echo "IMPORT PGDUMP 'nodelocal://0/import.sql" cockroach sql·
> ```
is this missing a pipe?


<!-- Sent from Reviewable.io -->

otan committed:
RFCs: update geospatial RFC

Update geospatial RFC to be in line with updated offerings and commands.

Release note: None

otan reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/52958)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @sumeerbhola)

___
*[docs/RFCS/20200421_geospatial.md, line 606 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/52958#-MF0lbWn057efgbgV9g1:-MF0x4zkDcoyXY4r1_j8:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/2a693e9ea985ca1b128be4a72e75e3345bb977b4/docs/RFCS/20200421_geospatial.md#L606)):*
<details><summary><i>Previously, sumeerbhola wrote…</i></summary><blockquote>

> ```
> echo "IMPORT PGDUMP 'nodelocal://0/import.sql" cockroach sql·
> ```
is this missing a pipe?
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->

otan commented:
bors r+
